src/main/java/org/fogbowcloud/saps/engine/core/archiver/Archiver.java

```diff
@@ -94,8 +93,8 @@ private void resetArchivingTasks() throws ArchiverException {
     private void deletePermData(SapsImage task) {
 
         try {
-            permanentStorage.delete(task);
-        } catch (PermanentStorageException e) {
+            permanentStorage.delete(task.getTaskId(), task.isFailed());
```

Comment:
please, add a FIXME comment in this class reporting we need to "think how to deal with errors during the data removal. today, we are logging, only. i guess that in some places we need to throw the exception, in other return a boolean flag:

Reply:
Done.